cmd/kubeadm/app/cmd/phases/join/controlplaneprepare.go

```diff
@@ -131,8 +131,26 @@ func runControlPlanePrepareManifestsSubphase(c workflow.RunData) error {
 		return err
 	}
 
-	// Generate missing certificates (if any)
-	return controlplane.CreateInitStaticPodManifestFiles(kubeadmconstants.GetStaticPodDirectory(), cfg)
+	fmt.Printf("[control-plane] Using manifest folder %q\n", kubeadmconstants.GetStaticPodDirectory())
+
+	components := []string{
```

Comment:
Don't define this. Use `kubeadmconstants.ControlPlaneComponents` instead.

Reply:
forgot about this. will update later today!